app/models/manageiq/providers/amazon/cloud_manager/virtual_template.rb

```diff
@@ -1,3 +1,4 @@
 class ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate < ::ManageIQ::Providers::CloudManager::VirtualTemplate
-  validates :cloud_subnet_id, :availability_zone_id, :ems_ref, :presence => true
+  validates :cloud_network_id, :availability_zone_id,
+            :ems_ref, :ems_id, :flavor_id, presence: true
```

Comment:
_unfortunately_ we prefer hash rockets
`:presence => true`

@Fryguy can we get miq-bot to comment on PRs on this repo?